.do/db-cluster/Makefile

```diff
@@ -0,0 +1,53 @@
+# terraform setup
```

Comment:
Why do we need a specific makefile for this one? Isn't it most of the stuff here just duplicated from the other one?
Can we have a single one under .doks?